addons/im_livechat/static/src/legacy/public_livechat.js

```diff
@@ -16,16 +16,8 @@ var WebsiteLivechatWindow = require('im_livechat.legacy.im_livechat.WebsiteLivec
 
 var _t = core._t;
 var QWeb = core.qweb;
+var { LIVECHAT_COOKIE_HISTORY, HISTORY_LIMIT, RATING_TO_EMOJI } = require('im_livechat.legacy.im_livechat.Constants');
```

Comment:
Move this LOC with the `im_livechat` imports above.
Follow alphabetical order by name of module.
So it should be just before `im_livechat.model`.